packages/components/src/templates/next/render/render.tsx

```diff
@@ -23,6 +23,7 @@ import {
   Video,
 } from "../components"
 import { LogoCloud } from "../components/complex/LogoCloud"
+import { ChildrenPages } from "../components/internal"
```

Comment:
issue(non-blocking): I think should move it out to become a `complex` block. The `internal` components are meant for those that do not have a JSON schema attached to them.

Reply:
ok sounds good!